packages/buefy-next/src/components/progress/Progress.spec.ts

```diff
@@ -1,32 +1,35 @@
 import { shallowMount } from '@vue/test-utils'
-import BProgress from '@components/progress/Progress'
+import type { VueWrapper } from '@vue/test-utils'
+import BProgress from '@components/progress/Progress.vue'
+import { describe, expect, it } from 'vitest'
+
+let wrapper: VueWrapper<InstanceType<typeof BProgress>>
 
 describe('BProgress', () => {
     it('is called', () => {
-        const wrapper = shallowMount(BProgress)
```

Comment:
Could you revert `const wrapper` → `let wrapper` changes? Although, these changes won't affect the outcome, I would like to separate refactoring from TypeScript migration. Here, especially, the original structure looks better.

Reply:
Sure! How would typing work in that case?

Reply:
You do not have to explicitly give a type, because TypeScript can infer it.

Reply:
> You do not have to explicitly give a type, because TypeScript can infer it.

Yeah I figured that out when I ran `npm run type-check`